edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -0,0 +1,463 @@
+    letter-spacing: 0.16em;
+  }
+
+  .course .details .course-actions .wrapper-action-more,
+  .course .details .course-actions .entitlement-actions-wrapper {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+  }
+
+  .course .details .course-actions .wrapper-action-more .action-more,
+  .course .details .course-actions .entitlement-actions-wrapper .action-more,
+  .course .details .course-actions .wrapper-action-more .entitlement-action-more,
+  .course .details .course-actions .entitlement-actions-wrapper .entitlement-action-more {
+    width: 100%;
+    font-size: 14px;
+    box-shadow: none;
+    background: $inverse;
+    text-shadow: none;
+    color: $primary;
+    border: none;
+  }
+
+  .course .details .course-actions
+  .wrapper-action-more div.actions-dropdown {
+      min-width: 108px;
+      display: none;
+    position: absolute;
+    top: 40px;
+    width: 100%;
+    pointer-events: none;
+  }
+
+  .wrapper-action-more .actions-dropdown
+  .actions-dropdown-list {
+      box-sizing: border-box;
+    display: table;
+    box-shadow: 0 1px 1px rgba(0,0,0,0.1);
+    position: relative;
+    width: 100%;
+    border-radius: 3px;
+    margin: 5px 0 0 0;
+    padding: 5px 10px;
+    background: #D4EAE4;
+    list-style: none;
+  }
+
+  .course .details .course-actions .wrapper-action-more
+  .actions-dropdown.is-visible{
+      display: block;
+    pointer-events: auto;
+  }
+  .course .details .course-actions .wrapper-action-more
+  .actions-dropdown .actions-dropdown-list {
+      box-sizing: border-box;
+    display: table;
+    box-shadow: 0 1px 1px rgba(0,0,0,0.1);
+    position: relative;
+    width: 100%;
+    border-radius: 3px;
+    margin: 5px 0 0 0;
+    padding: 5px 10px;
+    background: #D4EAE4;
+  }
+
+  //dashboard-subcription-mobile
+  @media (max-width: 991px){
+    .course .details {
+      padding-left: 10px;
+    }
+  }
+
+  @media (max-width: 767px){
+    .course .details .wrapper-course-image {
+      display: none;
+    }
+  }
+
+  @media (max-width: 767px){
+    .course .details .wrapper-course-details {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  @media (max-width: 767px){
+    .course .details .wrapper-course-details .course-info,
+    .course .details .wrapper-course-details .wrapper-course-actions {
+      flex-grow: 1;
+      text-align: left;
+    }
+  }
+
+  @media (max-width: 767px){
+    .course .details .wrapper-course-details
+    div.wrapper-course-actions div.course-actions {
+      padding-top: 50px;
+    }
+  }
+
+  @media (max-width: 767px){
+    .course .details .wrapper-course-details .course-info .wrapper-action-more,
+    .course .details .wrapper-course-details .course-info .entitlement-actions-wrapper,
+    .course .details .wrapper-course-details .wrapper-course-actions .wrapper-action-more,
+    .course .details .wrapper-course-details .wrapper-course-actions .entitlement-actions-wrapper {
+      padding-top: 3px;
+    }
+  }
```

```suggestion
  @include media-breakpoint-down(md) {
    .course .details .wrapper-course-image {
      display: none;
    }
  }

  @include media-breakpoint-down(md) {
    .course .details .wrapper-course-details {
      display: flex;
      flex-direction: column;
    }
  }

  @include media-breakpoint-down(md) {
    .course .details .wrapper-course-details .course-info,
    .course .details .wrapper-course-details .wrapper-course-actions {
      flex-grow: 1;
      text-align: left;
    }
  }

  @include media-breakpoint-down(md) {
    .course .details .wrapper-course-details
    div.wrapper-course-actions div.course-actions {
      padding-top: 50px;
    }
  }

  @include media-breakpoint-down(md) {
    .course .details .wrapper-course-details .course-info .wrapper-action-more,
    .course .details .wrapper-course-details .course-info .entitlement-actions-wrapper,
    .course .details .wrapper-course-details .wrapper-course-actions .wrapper-action-more,
    .course .details .wrapper-course-details .wrapper-course-actions .entitlement-actions-wrapper {
      padding-top: 3px;
    }
  }
```
Same suggestion about Bootstrap brakpoints